.github/workflows/build.yml

```diff
@@ -1,11 +1,9 @@
 name: Build
 on:
-  pull_request:
   push:
     branches:
       - main
 jobs:
-
```

Comment:
All changes below this are just formatting. It appears this file never went through a formatter.